test/metabase/models/user_test.clj

```diff
@@ -511,3 +512,13 @@
               (is (u.password/verify-password plaintext-password
                                               (salt)
                                               new-hashed-password)))))))))
+
+(deftest has-a-last-acknowledged-version
+  (testing "last-acknowledged-version can be read and set"
+    (mt/with-test-user :crowberto
+      (try
+        (is (nil? (setting/get :last-acknowledged-version)))
+        (setting/set! :last-acknowledged-version "47")
```

Comment:
maybe add a test to make sure the setting is user-local only?

Reply:
Good point, done